src/components/Connect/Burner/CreateSeed/index.stories.jsx

```diff
@@ -15,12 +15,10 @@ export default {
   parameters: { actions: { argTypesRegex: '^on.*' } }
 }
 
-const Template = args => {
-  return (
-    <>
-      <CreateSeed {...args}>{args.name}</CreateSeed>
-    </>
-  )
-}
+const Template = args => <CreateSeed {...args} />
 
 export const Base = Template.bind({})
+Base.args = {
+  back: () => console.log('Clicked Back'),
+  next: () => console.log('Clicked Next')
+}
```

Comment:
Missing props here that should be required. It didn't give an error before because the component's proptypes weren't set